Updated steps to turning on debug for integrations (#28664)
home-assistant/home-assistant.github.io: source/_docs/configuration/troubleshooting.markdown

### Enabling Debug Logging

> To enable debug logging for an integration, go to **Settings -> Devices & Services** and then click the integration. Click on the triple dots for the integration giving you trouble and click **Enable Debug Logging**.

Reviewer: ```suggestion
To enable debug logging for an integration, go to **Settings** > **Devices & Services** and select the integration. Select the triple dots for the integration giving you trouble and select **Enable Debug Logging**.
```
